product_docs/docs/biganimal/release/getting_started/identity_provider/index.mdx

```diff
@@ -1,12 +1,19 @@
 ---
 title: "Setting up your identity provider"
 navTitle: "Setting up your identity provider"
-description: Applies when you purchased BigAnimal directly from EDB
+description: How to set up your identify provider when you purchased BigAnimal directly from EDB
 ---
 
-If you purchased BigAnimal directly from EDB, you must set up your identity provider (IDP) before accessing BigAnimal for the first time. After setting up your identity provider, you can add users to BigAnimal by adding them to the designated group in your identity provider. Once you've logged into BigAnimal using your identity provider, you can set up your cloud service provider (CSP) in the BigAnimal portal to complete onboarding. 
+If you purchased BigAnimal through EDB, you can use EDB Okta as your identity provider. After signing in for the first time, you can then set up your own identity provider if you wish. 
 
-If you purchased through Azure Marketplace, BigAnimal authenticates users using Azure Active Directory (AD) and you don't have to complete these steps. Azure AD is linked during subscription.
+When using your own identity provider, you add users to BigAnimal by adding them to the designated group in your identity provider. Once you've logged into BigAnimal using your own identity provider, you can set up your cloud service provider in the BigAnimal portal to complete onboarding. 
+
+When using EDB Okta, you can invite users by selecting **Invite New User** on the Users page. After providing their EDB account email and their role, you can send them an invitation link. 
```

Comment:
no matter the user login by which IdP: `EDB Account` or `their own IdP`, the organization-owner, admin, and project-owner can invite users by .... ....